src/NLog/Targets/ChainsawTarget.cs

```diff
@@ -67,6 +67,7 @@ public class ChainsawTarget : NLogViewerTarget
         public ChainsawTarget()
         {
             IncludeNLogData = false;
+            OptimizeBufferReuse = GetType() == typeof(ChainsawTarget);
```

Comment:
Why not just true? Because of custom sub classes?

Reply:
Yes minimum amount of surprises possible.

Reply:
Please add this as code comments, thanks in advance!